### ISSUE TITLE
PgAdmin4 table backup includes "DROP DATABASE" without selecting drop database in options

### ISSUE THREAD

**Issue by Adventurer2021** [pgadmin-org/pgadmin4#5887]
pgAdmin4 V6.20 (as well as previous versions) backup wizard adds the "DROP DATABASE" command when creating a backup of a table (not entire DB) **without selecting the "DROP DATABASE switch**.  I've got about 86 tables that I've backed up using a file extension of "backup" for all my table backup files. 

When attempting to restore these backups on a Linux Mageia system **via a terminal command** 
Command:
**_pg_restore --host "localhost" --port "5432" --username "postgres" --no-password --dbname "myname" --create --clean --verbose "dumpfile"_**

getting error:

 _"pg_restore: error: could not execute query:
 ERROR:  cannot drop the currently open database"_

Using grep command:
 **_grep -sm 1 "DROP DATABASE" *backup_** 

I found that all my table backups had the "DROP DATABASE" statement in the dump file. I can restore the backed up dump files using pgAdmin restore wizard without issue on my Windows 11 Pro and Linux Mint systems using pgAdmin4 but another system running Linux Mageia is the issue. 

Would like to know why the pgAdmin4 GUI is adding this "DROP DATABASE" command **without flipping the "DROP DATABASE" switch** when I'm backing up a table, (not a full database) in the options panel of the backup wizard? 

1. Open pgAdmin4/right click on table name
2. Select backup and enter file name of choice through pop ups
3. Click "backup"
4. Wait for backup process to complete

**Expected behavior**
Creation of a backup (dump file)  is achieved but without "DROP DATABASE" command in dump file

**Error message**
When attempting to restore the table on the Linux Mageia system, the pg_restore terminal command (noted above) responds with:
 _"pg_restore: error: could not execute query:
 ERROR:  cannot drop the currently open database"_

**Desktop (please complete the following information):**
 - OS: Windows 11 Pro, Linux Mint, Linus Mageia (remote system)
 - Version: 10.0.22621 Build 22621
 - Mode: Desktop - Dell XPS8940
 - Browser: pgAdmin4 V6.20 
 - System default browser on Windows 11 Pro system which is doing the backups: Brave

Update: I found that running the pgAdmin4 backup of a table on my Linux Mint system also adds the "DUMP DATABASE" text to the dump file.

**Comment by pravesh-sharma** [comment]
Hi @Adventurer2021 

This seems not to be pgAdmin issue as pg_dump/pg_restore are utility tools provided by PostgreSQL which pgAdmin uses. However I tried reproducing the issue and below are my findings.

1. Took a backup of table using pg_dump from command line and that file also contains the "Drop Database" statement.
2. Tried to restore the table using pg_restore from command line I am getting the error you mentioned but the table is successfully restored.

Attached screenshot for reference.
<img width="1784" alt="Screenshot 2023-03-07 at 3 05 12 PM" src="https://user-images.githubusercontent.com/111979571/223383745-87bb0d05-d50f-421b-8b27-969d722fc8a7.png">
